integration-test/pom.xml

```diff
@@ -39,6 +39,12 @@
       <groupId>org.ehcache</groupId>
       <artifactId>ehcache-clustered</artifactId>
       <version>${ehcache.version.xml}</version>
+      <exclusions>
```

Comment:
When using a version range, we need to avoid transitive "pinning" of the version to prevent version convergence failures -- annoying but necessary.

Reply:
Isn't it right that this dependency pins the version? The ehcache libraries will pin the version... it's just that right now the Ehcache version is trying to pin it outside the allowable range?

Reply:
Could be.  When Ehcache is updated, this dependency _might_ be able to be updated to allow the pinning.  Will need to play with it.  (Logback 1.2.11 requires a minimum of SLF4J 1.7.32; Ehcache will need to have a comparable change.)